src/redux/selectedPokemonSlice.js

```diff
@@ -29,7 +29,7 @@ export default function selectedPokemonReducer(state = initialState, action) {
 export const fetchPokemonById = (id) => {
   return (dispatch) => {
     dispatch(fetchPokemonByIdRequest);
-    axios.get(`/${id}`).then((res) => {
+    axios.get(`pokemon/${id}`).then((res) => {
```

Comment:
probably you can group this function and the above one to one file, cause they are both working with pokemons

Reply:
Done